services/galley/src/Galley/API/LegalHold/Conflicts.hs

```diff
@@ -107,7 +115,12 @@ guardLegalholdPolicyConflictsUid self otherClients = do
       otherUids :: [UserId]
       otherUids = nub $ Map.keys . userClients $ otherClients
 
-  when (nub otherUids /= [self {- if all other clients belong to us, there can be no conflict -}]) $ do
+      -- if all other clients belong to us, or if there are no clients at all, there can be no
+      -- conflict.
+      conflictPossible :: Bool
+      conflictPossible = not . null $ nub otherUids \\ [self]
+
+  when conflictPossible $ do
```

Comment:
can you perhaps explain why we need this? I don't understand it :3

Reply:
i tried again.  if you still don't understand it, that's fair, but let's talk then and afterwards *you* write it down.  :)

Reply:
alright, thank you, will have a read.